### DISCUSSION TITLE
WT32-SC01-PLUS screen?

### DISCUSSION THREAD

**Discussion by dexterbot80** [AxxAxx/AxxSolder#32] [Ideas]
Hello, You have done an excellent job here.
How about the WT32-SC01-PLUS screen?
A very nice interface could be created with [this program](https://squareline.io/), for example, and we could make a station with 4 different outputs.
[https://www.youtube.com/watch?v=6uKf5Bj0xcc&t=377s](https://www.youtube.com/watch?v=6uKf5Bj0xcc&t=377s)
Do you think a little about this proposal? :)

**Comment by AxxAxx** [comment]
I have been thinking alot about screens... I would like to have a screen that is availible from several sources and possibly from a know supplier. 
I have been thinking about integrating this into next HW version: 
https://www.adafruit.com/product/4311

It is a color screen, FTF, resonable price and availible through Mouser, Digikey and other big suppliers.

What do you think?

**Reply by dexterbot80** [reply]
I also saw this display, but compared to the [WT32-SC01-PLUS](http://en.wireless-tag.com/product-item-26.html), it is small and not touch sensitive.
[WT32-SC01-PLUS](http://en.wireless-tag.com/product-item-26.html) also has that finished frame that allows you to integrate it very easily and it is based on ESP32-S3 which makes it quite fast - I think it can also play small movies :).
I would choose [WT32-SC01-PLUS](http://en.wireless-tag.com/product-item-26.html) because you can create an extremely beautiful user interface and I think it would give a touch of elegance to the soldering station.
Maybe in the future you update and add an extra outlet for a soldering iron or another tool... and a bigger display would help.
The final work would be very nice :). It is just my opinion and I accept any comment from a teacher like you. As I told you at the beginning, I really like what you have done so far.

**Reply by AxxAxx** [reply]
I think the WT32-SC01-PLUS screen does not fit my needs for AxxSolder, it is too big, too expensive and can can not be sourced from mouser/digikey. Thanks anyway. I do also not really like touch interfaces on devices like these.

**Reply by dexterbot80** [reply]
Honestly, I don't think it's very expensive or very big. I think that a soldering station equipped with such a display would be very elegant.
[https://www.alibaba.com/product-detail/Upgrade-WT32-SC01-plus-16MB-hmi_1600609718238.html?spm=a2700.shop_cp.16319.1.64004a46aheFyj](https://www.alibaba.com/product-detail/Upgrade-WT32-SC01-plus-16MB-hmi_1600609718238.html?spm=a2700.shop_cp.16319.1.64004a46aheFyj)
But you are the boss here and only you decide what is ok for your device :)
​
